cmd/auth-provider-gcp/apis/credentialprovider/README.md

```diff
@@ -0,0 +1 @@
+Temporarily copied from code that will land in k8s v1.20 - please do not modify!
```

Comment:
Should this read?:
Temporarily copied/modified from k8s v1.20 code that should land in k8s v1.21 . Please reach out to SIG Cloud Provider with any questions.

Reply:
This is from [k8s.io/kubelet/pkg/apis/credentialprovider](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kubelet/pkg/apis/credentialprovider), which I think is getting merged into v1.20 (it comes from https://github.com/kubernetes/kubernetes/pull/94196). I added the bit about the SIG though. Perhaps this could be temporarily be copied into `vendor/` instead of `cmd/auth-provider-gcp` or some additional README info added to make its origins more clear?

Reply:
Marking as resolved since this is code that has been merged into v1.20, and the README notes that the copied code can be removed once cloud-provider-gcp supports v1.20.